packages/analytics/README.md

```diff
@@ -1,5 +1,42 @@
+<!-- This block is inserted by scripts/copy-generic-readme.ts and not oclif like the commands block. Do not adjust it manually. --> 
+<!--sapCloudSdkLogo-->
+<a href="https://sap.com/s4sdk"><img src="https://help.sap.com/doc/2324e9c3b28748a4ae2ad08166d77675/1.0/en-US/logo-with-js.svg" alt="SAP Cloud SDK for JavaScript Logo" height="122.92" width="226.773"/></a>
+<!--sapCloudSdkLogoStop-->
+
 # cloud-sdk-analytics
 
 This package contains the code we use for collecting usage data.
 Please note that usage analytics is disabled by default, i.e. you need to explicitly opt-in for data to be collected.
-For more information on how to opt-in and which data is collected, please refer to [this document](https://github.com/SAP/cloud-sdk-cli/blob/main/usage-analytics.md).
+For more information on how to opt-in and which data is collected, please refer to [this document](https://github.com/SAP/cloud-sdk-js/blob/main/packages/analytics/usage-analytics.md).
+
+
+<!-- This block is inserted by scripts/copy-generic-readme.ts and not oclif like the commands block. Do not adjust it manually. --> 
+<!--sapCloudSdkCommonReadme-->
```

Comment:
What do you think about moving the comment into the generated block? Because currently I think you are adding it manually everywhere, which might be a bit annoying.
```suggestion
<!--sapCloudSdkCommonReadme-->
<!-- This block is inserted by scripts/copy-generic-readme.ts and not oclif like the commands block. Do not adjust it manually. --> 
```